docs/src/DeveloperDocumentation/new_developers.md

```diff
@@ -0,0 +1,80 @@
+# Introduction for new developers
+
+This document is meant to get new developers started. In general you have to do
+the following six steps for submitting changes to the Oscar source:
+
+1. Fork the main Oscar.jl repository. For this go to the [Oscar.jl github
+   page](https://github.com/oscar-system/Oscar.jl) and click on "Fork" at the
+   upper right.
+2. Clone your forked repository to your local machine.
+   ```
+   git clone git@github.com:your_github_username/Oscar.jl
+   ```
+3. Create a new branch, usually the naming convention is to use your initials
+   ("yi") and then describe your change, for example:
+   ```
+   git checkout -b yi/new_feature
+   git checkout -b yi/issue1234
+   git checkout -b yi/document_feature
+   ```
+4. Edit your source and try out your changes locally (see below). To use your local copy of
+   the sources, start Julia and
+   ```
+   ]dev /path/to/local/clone/of/your/fork/of/Oscar.jl
```

Comment:
This is exactly how I work, but I wonder if not others are instead following this approach:

1. Start Julia and do `]dev Oscar`
2. Then `cd ~/.julia/dev/Oscar`
3. Create branch... etc.
4. ...

Reply:
I mention this possibility now, but only stiefmuetterlich. I have great fear that I might tell such a poor soul to delete their `.julia` folder.